pkg/release/release.go

```diff
@@ -20,18 +23,22 @@ const (
 
 type Config struct {
 	ApprClient *apprclient.Client
+	FileLogger *filelogger.FileLogger
 	HelmClient *helmclient.Client
+	K8sClient  kubernetes.Interface
 	Logger     micrologger.Logger
 
-	Namespace string
+	TargetNamespace string
```

Comment:
What's the reason behing calling it a target namespace? For me `Namespace` makes total sense.

Reply:
Because this is the target of our installation of resources, I can revert back to namespace if you prefer.

Reply:
This is kind of obvious that this is the namespace where releases will be installed, isn't it?
I'd prefer to call it a `Namespace` yes, but no blocker.

Reply:
Reverted back to `namespace` PTAL.